### PR TITLE
Add `.gitignore` file and tweak script

### PR DESCRIPTION
**Pull request by tjkuson**
Add `.gitignore` to prevent files we would rather not be tracked from being tracked. Add doc strings and tweak the logic of the main script.

### CODE REVIEW

**Comment by tjkuson** [comment]
I also added a jumpscare.

File: names_exercise.py [hunk 1]
```diff
@@ -1,19 +1,26 @@
+"""Simple script to print names given as a list."""
```
Comment (BirchAD) [inline names_exercise.py:1]: Thanks!

Comment (tjkuson) [reply names_exercise.py:1]: You're welcome.

**Review by BirchAD — APPROVED** [review]
Looks good

**Pull request merged**